travis/publish.sh

```diff
@@ -1,9 +1,10 @@
 #!/usr/bin/env bash
-set -e
+set -ex
```

Comment:
The `x` is candidate for removal of course, but experience shows it'll be useful to have it there for the first few releases.